templates/huginn/0/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: '2'
+
+services:
```

Comment:
I would add load balancer service to publish the service, including a question about which port want to publish for it.
```
  huginn-lb:
    image: rancher/lb-service-haproxy:v0.7.15
    ports:
      - ${PUBLISH_PORT}:${PUBLISH_PORT}
```